From 656126c930dfb7c5eefd2fc1962fab178be86618 Mon Sep 17 00:00:00 2001
From: Christian Hergert <chergert@redhat.com>
Date: Thu, 6 Apr 2017 16:22:00 -0700
Subject: web-extension: fix NULL or empty string check

We need to compare against the character, not the pointer which
is done immediately to the left of this comparison.
---
 embed/web-extension/ephy-web-extension.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/embed/web-extension/ephy-web-extension.c b/embed/web-extension/ephy-web-extension.c
index 003f9ba..70971ef 100644
--- embed/web-extension/ephy-web-extension.c
+++ embed/web-extension/ephy-web-extension.c
@@ -1200,7 +1200,7 @@ handle_method_call (GDBusConnection       *connection,
     if (!web_page)
       return;
 
-    if (base_uri == NULL || base_uri == '\0') {
+    if (base_uri == NULL || *base_uri == '\0') {
       g_dbus_method_invocation_return_error (invocation, G_DBUS_ERROR, G_DBUS_ERROR_INVALID_ARGS,
                                              "Base URI cannot be NULL or empty");
       return;
-- 
cgit v0.12

